oap-server/analyzer/log-analyzer/src/main/java/org/apache/skywalking/oap/log/analyzer/dsl/spec/extractor/ExtractorSpec.java

```diff
@@ -291,12 +285,14 @@ public void slowSql(@DelegatesTo(SlowSqlSpec.class) final Closure<?> cl) {
         builder.setTimeBucket(timeBucketForDB);
         builder.setTimestamp(log.getTimestamp());
 
-        String entityId = serviceMeta.getEntityId();
         builder.prepare();
-        DatabaseSlowStatement databaseSlowStatement = builder.toDatabaseSlowStatement();
-        databaseSlowStatement.setDatabaseServiceId(entityId);
```

Comment:
@yswdqz I think `entityId` had been set in `builder.toDatabaseSlowStatement()`. Please confirm why this was required.

Reply:
It's right. I was worried that the IDs generated twice would be different before. But it probably doesn't need to be worried about.